src/App.jsx

```diff
@@ -19,9 +19,9 @@ export default function App() {
             <Route path="/login">
               <Login />
             </Route>
-            <Route path="/treehouse/members">
+            <PrivateRoute path="/treehouse/members">
```

Comment:
Nice job here creating context in your app to make the state accessible to every file in your app